src/components/pokemon-card/PokemonCard.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import './PokemonCard.css'
+import axios from 'axios'
+
+function PokemonCard({ pokemonName }) {
+    const [pokemon, setPokemon] = useState({});
```

Comment:
Hier ook, maar dan meer om makkelijker te kunnen checken of het al ingeladen is of niet
```suggestion
    const [pokemon, setPokemon] = useState({});
```